### PR TITLE
Introduce GitHub pull reviews api

### PR DESCRIPTION
**Pull request by haya14busa**
ref: #58

### CODE REVIEW

File: github.go [hunk 1]
```diff
@@ -157,3 +197,57 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
...
+
+// ---
+// GitHub PullRequest Review API Implementation
+// ref: https://github.com/google/go-github/issues/495
+
+const (
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
+)
+
+type Review struct {
```
Comment (haya14busa) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

File: github.go [hunk 2]
```diff
@@ -157,3 +197,57 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
...
+	Body     *string          `json:"body,omitempty"`
+	Event    *string          `json:"event,omitempty"`
+	Comments []*ReviewComment `json:"comments,omitempty"`
+}
+
+type ReviewComment struct {
```
Comment (haya14busa) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

File: github.go [hunk 3]
```diff
@@ -157,3 +201,57 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
...
+
+// ---
+// GitHub PullRequest Review API Implementation
+// ref: https://github.com/google/go-github/issues/495
+
+const (
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
+)
+
+type Review struct {
```
Comment (haya14busa) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

File: github.go [hunk 4]
```diff
@@ -157,3 +201,57 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
...
+	Body     *string          `json:"body,omitempty"`
+	Event    *string          `json:"event,omitempty"`
+	Comments []*ReviewComment `json:"comments,omitempty"`
+}
+
+type ReviewComment struct {
```
Comment (haya14busa) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

File: github.go (hunk 3 above)
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

File: github.go (hunk 4 above)
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

File: .drone.sec [hunk 5]
```diff
@@ -1 +1 @@
-eyJhbGciOiJSU0EtT0FFUCIsImVuYyI6IkExMjhHQ00ifQ.jNZPGtoP-mdDdVeX3QoNjHkzW3hgBRL_HQKQRkC5dksDXBu9xOCH2dnrFVtPMMXlxdjBNAJw6bc3q27FZxgvBLqiIwj-x_gs5y_X7C5t11d3lWP2-bS-JbHpWiZ5t-Mh_lT-w0Tz4fPH9FtxdOERg_cYfFzXOLUNmq4G3lDFQbLy8Kx3MJNqnDgd3psTXAaJUOToII-wtBgtze-TbDb34kJtCrlEJ5-guepsRXXWl3uu8u7t8jvEgsrwp1rs09rdwGucb2dWf_vo5258pjud2-oinL2nzCKIdzXR4de5B-CFStm9D8QmLqoNOreWe05A2bS6tvsXLJPSOduFB_uH2A.IuaZb-A6fGgzcyff.3uGTMjhUXBGz7KjJGEDsoFar_EzpBb4U3geXkekhCoQoxsXbcWDWcCJLiuX3lXM8aeA6wZHgLpr_6dNGjO10UpwyRozzenzA1dJ3YIqT_PjIWn4sVlOUBT2oIOn7EyCh-2JJWghl9Jnsf7K1_a-fxfLzqB4NrkUHdJNaev5gvh2m85SrfIprKIjI6viT8lLMFtpTsPbmK56-J9Zsgxg-.toQoDcioYYq9TZ6yNiAtug
```
Comment (reviewdog) [inline .drone.sec]: review test (manual)

File: github.go [hunk 6]
```diff
@@ -1,11 +1,19 @@
 package reviewdog
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

File: github.go [hunk 7]
```diff
@@ -157,3 +208,45 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
...
+
+// ---
+// GitHub PullRequest Review API Implementation
+// ref: https://github.com/google/go-github/issues/495
+
+const (
+	mediaTypePullRequestReview = "application/vnd.github.black-cat-preview+json"
+)
+
+type Review struct {
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Review should have comment or be unexported

File: github.go [hunk 8]
```diff
@@ -157,3 +208,45 @@ func (g *GitHubPullRequest) comment() ([]*github.PullRequestComment, error) {
...
+	Body     *string          `json:"body,omitempty"`
+	Event    *string          `json:"event,omitempty"`
+	Comments []*ReviewComment `json:"comments,omitempty"`
+}
+
+type ReviewComment struct {
```
Comment (reviewdog) [inline github.go]: **[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type ReviewComment should have comment or be unexported

File: github.go (hunk 7 above)
Comment (haya14busa) [inline github.go]: it's ok because this struct is temporary one and replace it with go-github implementation later.

**Pull request merged**